library/globals.inc.php

```diff
@@ -1232,6 +1232,41 @@
       xl('Number of Appointments to display in the Onsite Patient Portal')
     ),
 
+    'appt_display_sets_option' => array(
+      xl('Appointment Display Sets - Ignore Display Limit (Last Set)'),
+      'bool',                           // data type
+      '1',                              // default
+      xl('Appointment Display Sets - Allow Last Set to Expand Past Appointment Display Limit')
+    ),
+
+    'appt_display_sets_color_1' => array(
+      xl('Appointment Display Sets - Color - Set One'),
+      'color_code',
+      '#FFFFFF',
+      xl('Appointment Display Sets - Color - Set One')
+    ),
+
+    'appt_display_sets_color_2' => array(
+      xl('Appointment Display Sets - Color - Set Two'),
+      'color_code',
+      '#E6E6FF',
+      xl('Appointment Display Sets - Color - Set Two')
+    ),
+
+    'appt_display_sets_color_3' => array(
+      xl('Appointment Display Sets - Color - Set Three'),
+      'color_code',
+      '#E6FFE6',
+      xl('Appointment Display Sets - Color - Set Three')
+    ),
+
+    'appt_display_sets_color_4' => array(
+      xl('Appointment Display Sets - Color - Set Four'),
+      'color_code',
+      '#FFE6FF',
+      xl('Appointment Display Sets - Color - Set Four')
+    ),
```

Comment:
I have rewritten the descriptions and will add them...

Color for odd sets (except when the last set is odd and the number of appointments scheduled exceeds the appointment display limit)

Color for even sets (except when the last set displayed is even and the number of appointments scheduled exceeds the appointment display limit)

Color for the last set displayed when the number of appointments scheduled exceeds the display limit and all appointments for the associated date are shown

Color for the last set displayed when the number of appointments scheduled exceeds the display limit and not all appointments for the associated date are shown